packages/app/src/dimension-mapper/SlicingSlider.tsx

```diff
@@ -17,23 +17,16 @@ function SlicingSlider(props: Props) {
   const { dimension, maxIndex, initialValue, onChange } = props;
 
   const [value, setValue] = useState(initialValue);
-
-  const [containerSize, containerRef] = useMeasure<HTMLDivElement>();
-  const sliderRef = useRef<ReactSlider>(null);
-
   const onDebouncedChange = useDebouncedCallback(onChange, [onChange], 250);
 
-  useEffect(() => {
-    sliderRef.current?.resize();
-  }, [containerSize]);
+  const [containerSize, containerRef] = useMeasure<HTMLDivElement>();
```

Comment:
We still need to measure the container to know whether to render marks on the slider.